ansible/group_vars/wakuv2-test.yml

```diff
@@ -47,6 +47,9 @@ nim_waku_websocket_ssl_key: '/etc/letsencrypt/live/{{ nim_waku_websocket_domain
 
 # Waku rln-relay parameters
 nim_waku_rln_relay_content_topic: '/toy-chat/2/luzhou/proto'
+nim_waku_eth_account_address: '0000000000000000000000000000000000000000'
```

Comment:
Shouldn't it be `nim_waku_rln_relay_eth_account_address`?

Reply:
Will be fixed in the future with - https://github.com/status-im/nwaku/issues/1196

Reply:
Well, if it's going to be fixed in `nim-waku` then we might as well use correct variable name in Ansible from the start.

Reply:
@rymnc is actively working on the issue, and we will reflect that change in this PR as well

Reply:
The naming has changed, now the PR is ready for review.